HeroVault/HeroVault/Features/Home/HomeScreen/CharacterModel.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CharacterModel.swift
+//  HeroVault
+//
+//  Created by Gia Catano on 2024/04/20.
+//
+
+import Foundation
+
+// MARK: Marvel Data Protocol
+
+protocol MarvelData {
+    var id: Int { get }
+    var name: String { get }
+    var overview: String { get }
+    var thumbnail: String { get }
+    var isFavourited: Bool { get }
+}
+
+// MARK: Character Response Model
+
+struct CharacterResponse: Codable {
+    let data: CharacterData
+}
+
+struct CharacterData: Codable {
+    let results: [Character]
+}
+
+struct Character: Codable, MarvelData {
+    let id: Int
```

Comment:
```suggestion
    let characterID: Int
```
Remember to add coding key

Reply:
same as below